IndicoV2.StrawberryShake/schema.extensions.graphql

```diff
@@ -0,0 +1 @@
+extend schema @key(fields: "id")
```

Comment:
no idea, It's a generated code (`./init.sh`  - so it might be downloaded from the indico server?).

usually it's used to mark fields as a `key` for the entity (fields we can index with), but looks weird in this context.

[Sample usage](https://docs.amplify.aws/cli/graphql-transformer/key#how-to-use-key)
```
type Customer @model @key(fields: ["email"]) {
  email: String!
  username: String
}
```

Reply:
ahh makes sense, yeah it was a little confusing so I wasn't sure if that was added by you or automatically